src/xunit.console.netcore/Program.cs

```diff
@@ -298,8 +298,8 @@ static XElement ExecuteAssembly(object consoleLock, string defaultDirectory, Xun
                     {
                         lock (consoleLock)
                         {
-                            Console.ForegroundColor = ConsoleColor.Red;
-                            Console.WriteLine("ERROR:       {0} has no tests to run", Path.GetFileNameWithoutExtension(assembly.AssemblyFilename));
+                            Console.ForegroundColor = ConsoleColor.Yellow;
+                            Console.WriteLine("Warning:       {0} has no tests to run", Path.GetFileNameWithoutExtension(assembly.AssemblyFilename));
                             Console.ForegroundColor = ConsoleColor.Gray;
```

Comment:
What cases do you see as being valid for not having any tests to run? If a test project doesn't have any tests to run we should avoid running it at the project level.